### PR TITLE
inital commit of progressive delivery w dynatrace tutorial

### PR DESCRIPTION
**Pull request by grabnerandi**

### CODE REVIEW

File: site/tutorials/snippets/install/cluster-aks.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ We are going to setup a Kubernetes cluster in Azure. Therefore, please download
   - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 1. Create AKS cluster
-  - Master version >= `1.15.x` (tested version: `1.15.10`)
+  - Master version >= `1.15.x` (tested version: `1.15.5`)
   - Size of the cluster: One **D8s_v3** node
 
-Find a full compatibility matrix for supported Kubernetes versions [here](https://keptn.sh/docs/0.6.0/installation/k8s-support/).
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-aks.md:20]: please use latest version of this file to include this note

File: site/tutorials/snippets/install/cluster-eks.md [hunk 2]
```diff
@@ -15,12 +15,19 @@ We are going to setup a Kubernetes cluster in EKS.
   - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) (version >= 1.16.156)
 
 1. Create EKS cluster on AWS. You can do so either via the online portal of AWS or via `eksctl`.
-  - version 1.15 (tested version: 1.15)
+  - version >= `1.13`, version >= `1.14` recommended (tested version: `1.14`)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-eks.md:18]: please use latest version of this file to use EKS version 1.15

File: site/tutorials/snippets/install/cluster-gke.md [hunk 3]
```diff
@@ -16,22 +16,20 @@ We are going to setup a Kubernetes cluster in the Google Cloud Platform.
   - [python 2.7](https://www.python.org/downloads/release/python-2716/) (required for Ubuntu 19.04)
 
 2. Create GKE cluster
-  - Master version >= `1.15.x` (tested version: `1.15.9-gke.22`)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-gke.md:19]: please use latest version of this file to use newer GKE version

File: site/tutorials/snippets/install/cluster-minikube.md [hunk 4]
```diff
@@ -32,4 +32,3 @@ Negative
   minikube tunnel 
   ``` 
 
-Find a full compatibility matrix for supported Kubernetes versions [here](https://keptn.sh/docs/0.6.0/installation/k8s-support/).
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-minikube.md:35]: please use latest version of this file to include this note

File: site/tutorials/snippets/install/cluster-openshift.md [hunk 5]
```diff
@@ -67,6 +67,3 @@ else
     exit
 fi
     ```
-
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-openshift.md:70]: please use latest version of this file to include this note

File: site/tutorials/snippets/install/cluster-pks.md [hunk 6]
```diff
@@ -22,15 +22,14 @@ Pivotal Container Service (PKS)
...
     HOST_NAME=host_name
     PLAN=small
     ```
 
-    ```
+    ```console
     pks create-cluster $CLUSTER_NAME --external-hostname $HOST_NAME --plan $PLAN
     ```
 
-Find a full compatibility matrix for supported Kubernetes versions [here](https://keptn.sh/docs/0.6.0/installation/k8s-support/).
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/cluster-pks.md:36]: please use latest version of this file to include this note

File: site/tutorials/snippets/install/download-keptnCLI.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ There are multiple options how to get the Keptn CLI on your machine.
   This will download and install the Keptn CLI automatically.
 
 -  Another option is to manually download the current release of the Keptn CLI:
-  1. Download the version for your operating system from [Download CLI](https://github.com/keptn/keptn/releases/tag/0.6.2)
+  1. Download the version for your operating system from [Download CLI](https://github.com/keptn/keptn/releases/tag/0.6.1)
```
Comment (jetzlstorfer) [inline site/tutorials/snippets/install/download-keptnCLI.md:17]: please use latest version of this file to have Keptn 0.6.2 installed

**Comment by grabnerandi** [comment]
handled this through different branch and pull request

**Pull request closed without merging**